Heading/Heading.module.less

```diff
@@ -0,0 +1,86 @@
+// Heading.module.less
+//
+@import "../styles/variables.less";
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.heading {
+	.applySkins({
+		border-bottom: @agate-heading-border-width solid transparent;
+		margin: 0 @agate-spotlight-outset (1.5 * @agate-spotlight-outset) @agate-spotlight-outset;
+		padding: @agate-heading-padding;
+		min-height: (1.5 * @agate-spotlight-outset);
+		letter-spacing: @agate-heading-letter-spacing;
+		line-height: @agate-heading-line-height;
+		color: @agate-heading-text-color;
+
+		&.showLine {
+			border-bottom-color: @agate-heading-border-color;
+			padding: @agate-heading-showline-padding;
+		}
+
+		&.title {
+			font-family: @agate-heading-title-font-family;
+			font-weight: @agate-heading-title-font-weight;
+			font-style: @agate-heading-title-font-style;
+			font-size: @agate-heading-title-font-size;
+			position: relative;
+			padding: @agate-heading-title-padding;
+
+			&::after {
```

Comment:
Suggest we guard this with `showLine` as well for consistency across the sizes.

```less
&.showLine {
  border-bottom-color: transparent;

  &::after {
    /* rest of the rules */
  }
}
```

Reply:
I'd considered that too. I was thinking that the "line" was separate from the "decoraton", and you may want both. I didn't see any designs in the doc that had a "page title" level header that did not include a decoration (the little colored block), but also figured that since the "showLine" implied a known entity, like a line that went all the way across the space, like an HR, that they should be interpreted separately. I can conceede this point though as both arguments are logical. Having the component behave more intuitively to a new user (rather than the author) is higher priority.

Reply:
The `showLine` prop is false by default, being a boolean. The decoration part should be showing by default for title-sized Headings. I'm really inclined to leave these two separate for the time being.